.gitignore

```diff
@@ -31,3 +31,4 @@ docs/modules.rst
 
 # IDEs
 .idea/
+*.iml
```

Comment:
Also, in general I disagree with putting your personal choice of IDE-generated files into our project gitignore. The proper process is to edit the `.gitignore_global` in your home directory or startup directory to contain whatever editors you use (I use vim, so all my `.swp` files get ignored from the globals file). Git picks up these global ignores from that file and prevents it from getting into your projects. If everyone listed their IDE files in our project, the gitignore would be like hundreds of lines long.

The only exception to this would be something the project uses like the py.test `.coverage/`directory, etc.

Reply:
@fubuloubu You bing up a good point.  I hadn't thought about minimizing the `.gitignore` before by not customizing to specific IDE's.  Do any `IDE`'s use the local `.gitignore` (vs `.gitignore_global`) for something special?

Reply:
It works like a Cascade, git first checks out your global and then appends the entries from the local git repo root, and finally from the folder down, if you only need it in a specific directory.

You can have your settings live in either place, but the best rule of thumb is to have your personal preferences stored in your global, and project specific entries live in the repo root .gitignore

Reply:
I had no idea about `.gitignore_global`! I'll remove that change and update my global gitignore.

Reply:
Awesome! If you could also remove that one for `.idea/` and the comment above, that would be great. No idea why I didn't reject that addition at the time.